pluggy.py

```diff
@@ -581,7 +595,7 @@ def varnames(func):
         try:
             func = func.__init__
         except AttributeError:
-            return ()
+            return (), ()
```

Comment:
@RonnyPfannschmidt you got a point, we should increase the version to `0.5.0` and mention this change in the CHANGELOG.

In addition, should we make this method private and remove it from the public API?